lib/reporting/identity_verification_report.rb

```diff
@@ -337,23 +382,40 @@ def query
         ),
         idv_final_resolution: quote(Events::IDV_FINAL_RESOLUTION),
         fraud_review_passed: quote(Events::FRAUD_REVIEW_PASSED),
+        fraud_event_names: quote(
+          [
+            Events::FRAUD_REVIEW_PASSED,
+            Events::FRAUD_REVIEW_REJECT_AUTOMATIC,
+            Events::FRAUD_REVIEW_REJECT_MANUAL,
+          ],
+        ),
+        normalized_fraud_review_pending: "coalesce(#{[
+          # rubocop:disable Layout/LineLength
+          'properties.event_properties.fraud_review_pending',
+          'not isblank(properties.event_properties.fraud_pending_reason)',
+          'properties.event_properties.fraud_check_failed',
+          '(ispresent(properties.event_properties.tmx_status) and properties.event_properties.tmx_status not in ["threatmetrix_review", "threatmetrix_reject"])',
+          '0',
+          # rubocop:enable Layout/LineLength
+        ].join(", ")})",
```

Comment:
Different events have different ways of tracking "fraud review pending," it turns out.

Reply:
What do you think about making this a method? With complex SQL queries I've used the pattern of deconstructing things into smaller methods to make things easier to maintain. I wonder if that might help make this method a little bit less scary?